lean4-infoview/package.json

```diff
@@ -25,7 +25,6 @@
   },
   "dependencies": {
     "@lean4/infoview-api": "^0.1.0",
-    "@tippyjs/react": "^4.2.6",
```

Comment:
The package-lock.json also needs updating with this.

Reply:
Indeed, thanks!